mw-webapp/src/component/sidebar/SidebarContent/SidebarContent.module.scss

```diff
@@ -5,16 +5,19 @@ $sidebar-padding: 25px;
 $sidebar-width: 300px;
 $sidebar-button-size: 20px;
 $sidebar-button-offset: 5px;
+$high-z-index: 100;
```

Comment:
lets put it in global variables and call `sidebarZIndex` because the nature of z indexes are global

Reply:
ok) done